docs/book/intro.md

```diff
@@ -38,14 +38,18 @@ you will need to do the following:
     ```php
     // config/autoload/dependencies.global.php:
     return [
-        'dependencies' => [
+        '<container config identifier>' => [
             'factories' => [
                 MyNamespace\Command\MyCommand::class => MyNamespace\Command\MyCommandFactory::class,
             ],
         ],
     ];
     ```
 
+    > ### Container Config Identifier
```

Comment:
You will get rid of the markdown issues by adding an empty line here.
Putting a newline here with `>` would already achieve that.

Reply:
@brian-n-millar 
Unfortunately, the documentation is a bit misleading here because you do not need step 3. and 4. if you use a config provider class in your laminas-mvc module.
And this additional explanation does not really help here. There must be a clear separation for Mezzio, a laminas-mvc based application and others.

At the moment, unfortunately, I don't know what is meant by "components" either:

> ## Integrating in Components
> 
> If you'd like to add integration with laminas-cli into your components…

https://docs.laminas.dev/laminas-cli/intro/#integrating-in-components

---

There more problems on this page:

- it is called "Introduction" but contains the usage
- the installation belongs on its own page
- the steps to create a command is incomplete and should be under its own heading
- the basic usage should follow an example, including the call via the console

Reply:
@froschdesign 

I agree that the current documentation might be miss-structured but I think we should re-structure this in a dedicated issue. 
Since the documentation does actually only provide details for mezzio, what would be your suggestion on how to split this into pieces without totally re-structuring the whole intro file?